napari/utils/misc.py

```diff
@@ -25,8 +25,12 @@ def running_as_bundled_app() -> bool:
     """Infer whether we are running as a briefcase bundle"""
     # https://github.com/beeware/briefcase/issues/412
     # https://github.com/beeware/briefcase/pull/425
+    if sys.modules['__main__'].__spec__.name in {"cProfile", "yappi"}:
+        # check if run from profiler
+        return False
     app_module = sys.modules['__main__'].__package__
     metadata = importlib_metadata.metadata(app_module)
```

Comment:
less conservative way is to change (will work with other profilers, but may hide some unexpected fail)
```suggestion
	try:
	    metadata = importlib_metadata.metadata(app_module)
	except importlib_metadata.PackageNotFoundError:
		return false
```

Reply:
@Czaki I kind of like the generality of this alternate solution... Anyone else at @napari/core-devs have an opinion?

Either way, could you also add a link to this PR so that the discussion is discoverable for someone reading the code?

Reply:
I'm afraid I don't really have enough experience here to weigh in, but curious what others think. Agree a link to this PR might be good

Reply:
The simpler (less conservative) way seems fine to me.  Chances that we _are_ running as a briefcase bundle but get some unexpected error that then returns False are very low

Reply:
Sounds like people prefer the less conservative approach @Czaki. If you are ok with committing this suggestion do you want to that and then I can merge?

Reply:
updated